pkg/sql/crdb_internal.go

```diff
@@ -9205,7 +9205,19 @@ var crdbInternalLDRProcessorTable = virtualSchemaTable{
 	schema: `
 CREATE TABLE crdb_internal.logical_replication_node_processors (
 	stream_id INT,
-	consumer STRING
+	consumer STRING,
+	recv_wait INTERVAL,
+	last_recv_wait INTERVAL,
+	flushes INT,
+	flush_time INTERVAL,
+	last_flush_time INTERVAL,
+	last_flush_kvs INT,
+	last_flush_bytes INT,
+	last_flush_slowest_batch INTERVAL,
+	cur_flush_running INTERVAL,
+	cur_flush_kvs INT,
+	cur_flush_processed INT,
```

Comment:
nit: i think this reduces the width of table and makes the col names slightly clearer:
s/cur_flush_kvs/cur_flush_total_kvs/r
s/cur_flush_processed/cur_flush_kvs/r

then it's clearer that that cur_flush_kvs and cur_flush_batches are running sums.